cleanlab/datalab/internal/issue_manager/regression/label.py

```diff
@@ -38,6 +39,10 @@ class RegressionLabelIssueManager(IssueManager):
     clean_learning_kwargs :
         Keyword arguments to pass to the :py:meth:`CleanLearning <cleanlab.regression.learn.CleanLearning>` constructor.
 
+    threshold :
+        The threshold to use to determine if an example has a label issue. Only used if
```

Comment:
this is a multiplier right not an absolute threshold

Reply:
Correct. The docstring has been updated.